specs/themes/ThemeSettings.md

```diff
@@ -0,0 +1,157 @@
+ThemeSettings 
+===
+
+# Background
+_This section is only for internal use, it should not be part of the public documentation._
+
+In UWP, apps can use the
+[AccessibilitySettings](https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.accessibilitysettings?view=winrt-22621)
+API to query the system's high-contrast setting, and detect when it changes.  But in a Win32/non-UWP context, the
+**AccessibilitySettings.HighContrastChanged** event never fires.  This is because the event relies on the **CoreWindow**
+object to handle the `WM_THEMECHANGED` message and fire an internal event to the **AccessibilitySettings** object -- but
+in a non-UWP, Win32/WinAppSDK context, no **CoreWindow** exists.
+
+The API described in this doc, **ThemeSettings**, is a WinAppSDK WinRT API that provides the functionality of
+**AccessibilitySettings**, but for a Win32 environment.  We expect to add other APIs to this type over time.
```

Comment:
Other uses I see for 'theme' include ApplicationTheme, ElementTheme, ShareUITheme all of which are about light mode / dark mode. To my eye it looks odd to have a ThemeSettings and only accessibility / high contrast settings on it. Is the expected APIs to add to this over time include the notion of theme used elsewhere?

Reply:
The expectation is to add more theme APIs here that we should have full APIs for:
* We need an API for light/dark mode, which currently you have to read from the registry
* System colors currently exposed by [GetSysColor](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getsyscolor) (Win32)
* Accent colors currently exposed by [UISettings.GetColorValue](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.uisettings.getcolorvalu)

Reply:
Added notes to background section.